website/views/shared/tasks/task.jade

```diff
@@ -132,24 +138,20 @@ li(bindonce='list', bo-id='"task-"+task.id', ng-repeat='task in obj[list.type+"s
               span.glyphicon.glyphicon-tasks
               span=env.t('addChecklist')
         form.checklist-form(ng-if='task.checklist')
-          fieldset.option-group(ng-if='!$state.includes("options.social.challenges")')
```

Comment:
It would be best to leave the fieldset as it was, both for HTML standardisation and to allow the `ng-if` to take effect.